tests/test_docs/test_orm_integration/test_orm_integration.py

```diff
@@ -152,18 +153,21 @@ def test_orm_integration_docs_example(self):
         seller_stategy_path = Path(
             seller_aea_name, "skills", "thermometer", "strategy.py",
         )
-        self.replace_file_content(seller_stategy_path, ORM_SELLER_STRATEGY_PATH)
+        self.replace_file_content(ORM_SELLER_STRATEGY_PATH, seller_stategy_path)
         self.fingerprint_item(
             "skill", "{}/thermometer:0.1.0".format(self.author),
         )
         self.run_install()
 
         # add non-funded key
         self.generate_private_key(COSMOS)
+        self.generate_private_key(COSMOS, COSMOS_PRIVATE_KEY_FILE_CONNECTION)
         self.add_private_key(COSMOS, COSMOS_PRIVATE_KEY_FILE)
-        self.add_private_key(COSMOS, COSMOS_PRIVATE_KEY_FILE, connection=True)
+        self.add_private_key(
+            COSMOS, COSMOS_PRIVATE_KEY_FILE_CONNECTION, connection=True
+        )
         self.replace_private_key_in_file(
-            NON_FUNDED_COSMOS_PRIVATE_KEY_1, COSMOS_PRIVATE_KEY_FILE
+            NON_FUNDED_COSMOS_PRIVATE_KEY_1, COSMOS_PRIVATE_KEY_FILE_CONNECTION
         )
```

Comment:
By using a different key for the connection and the agent we can rotate the key on the agent and avoid clashes when registering on SOEF